src/app/[locale]/seller/sale-items/[id]/page.tsx

```diff
@@ -150,7 +150,7 @@ export default function BuyFromSellerForm({ params }: { params: { id: string } }
             <p className="text-sm">
               {t('SCREEN.BUY_FROM_SELLER.REVIEWS_SCORE_MESSAGE', {seller_review_rating: sellerShopInfo.average_rating.$numberDecimal})}
             </p>
-            <Link href={`/seller/reviews/${sellerId}?buyer=true&seller_name=${sellerShopInfo.name}`}>
+            <Link href={`/seller/reviews/${sellerId}?buyer=true&user_name=${sellerShopInfo.name}`}>
```

Comment:
I've been wondering.. is it okay passing the user's `pi_uid` value openly like this from a security perspective..?  🤞

Reply:
Posed this question to Tate and pending his response.

Reply:
the only option is to encrypt the id but it has to be passed to identify the specific seller. I will be waiting.